scripts/packing_similarity_dendrogram/packing_similarity_dendogram.py

```diff
@@ -244,10 +250,9 @@ def main(input_file, matrix_file, n_ps_mols, output_ps_results, conf_threshold,
     ps.settings.packing_shell_size = n_ps_mols
     ps.settings.angle_tolerance = ps_angles
     ps.settings.distance_tolerance = ps_distances
-
     refcodes = []
 
-    input_name = input_file.rsplit(".")[0]
+    input_name = input_file.rsplit("\\")[-1].rsplit(".")[0]
```

Comment:
Is this to handle the paths of files? 
I think a better approach would be to use `os` otherwise this won't work on Linux 
```python
input_name = os.path.basename(input_file).split(".")[0]
```